### ISSUE TITLE
Incorrect aria label on datepicker buttons

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#459]
### Describe the bug

All buttons should have an accessible name, in the datepicker component, several of the aria-labels are placed directly onto the p-button tag instead of being passed into the component to be placed on the actual button element.

### Pull Request Link

https://github.com/primefaces/primeng/pull/1153

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/datepicker#time

### Environment

Can be reproduced on any environment running Angular 19  with PrimeNG 19.

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

Inspect .p-datepicker-increment-button within https://primeng.org/datepicker#time datepicker overlay panel and noticed the button tags don't have aria-label set, instead the p-button tag wrapping it has the aria-label

### Expected behavior

All .p-datepicker-increment-button have an accessible name on them (they are icon only buttons which require aria-label be set to pass accessibility)

> Migrated from `primefaces/primeng#18405` — originally by `@cvillella` on 2025-05-25

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-11-22T00:19:20.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!

**Comment by geromegrignon** [comment]
*[Originally posted by `@Platacoder` on 2026-03-17T20:05:30.000Z]*

This is still an issue!
